parentchain/Cargo.toml

```diff
@@ -0,0 +1,100 @@
+[package]
+name = "pallet-parentchain"
+description = "The remote attestation registry and verification pallet for integritee blockchains and parachains"
+version = "0.9.0"
+authors = ["Integritee AG"]
+homepage = "https://integritee.network/"
+repository = "https://github.com/integritee-network/pallet-teerex/"
+license = "Apache-2.0"
+edition = "2018"
+resolver = "2"
+
+[dependencies]
+serde = { features = ['derive'], optional = true, version = '1.0.101' }
+log = { version = "0.4.14", default-features = false }
+hex-literal = { version = "0.3.2", optional = true }
+
+[dependencies.codec]
+default-features = false
+features = ['derive']
+package = 'parity-scale-codec'
+version = '2.0.0'
+
+[dependencies.frame-support]
+default-features = false
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+[dependencies.frame-system]
+default-features = false
+package = 'frame-system'
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+[dependencies.sp-core]
+default-features = false
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+[dependencies.sp-io]
+default-features = false
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+
+[dependencies.sp-runtime]
+default-features = false
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+[dependencies.sp-std]
+default-features = false
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+[dependencies.frame-benchmarking]
+default-features = false
+optional = true
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+[dev-dependencies.sp-keyring]
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+[dev-dependencies.pallet-balances]
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "4.0.0-dev"
+
+
+[dev-dependencies]
+hex-literal = "*"
+env_logger = "0.7.1"
+log = "*"
+
+[features]
+default = ['std']
+std = [
+    'serde',
+    'codec/std',
+    'frame-support/std',
+    'frame-system/std',
+    'sp-core/std',
+    'sp-io/std',
+    'sp-runtime/std',
+    'sp-std/std',
+]
+
+runtime-benchmarks = [
+    'frame-benchmarking',
+    "hex-literal"
```

Comment:
We have not a definite way of using ` '` or `"`, but lets stay consistent within the same file at least. I've looked around a little bit, and I think the prefered way is to use `"`

Reply:
We actually have consensus to use `"` but it might not be consistent yet.

Reply:
Oho, good to know about that. Mabye we should write a StyleGuide @gaudenzkessler 😆